can-control_test.js

```diff
@@ -480,6 +480,16 @@ test("Creating an instance of a named control without passing an element", funct
 	ok(myControlInstance.element.className === 'MyControl', "Element has the correct class name");
 });
 
+test("Creating an instance of a named control passing a selector", function() {
+
+	this.fixture.appendChild( fragment('<div id=\'my-control\'>d</div>') );
+	
+	var MyControl = Control.extend('MyControl');
+	var myControlInstance = new MyControl('#my-control');
+
+	ok(myControlInstance.element.className === 'MyControl', "Element has the correct class name");
```

Comment:
In IE9, the `className` gets set as `" MyControl"`, so you should use `className.has.call(myControlInstance.element, 'MyControl')`.